cats-and-dogs/src/components/Dogs/RandomImg.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import axios from "axios";
+import DisplayDog from "./DisplayDog"
+
+class Random extends React.Component{
+   
+    state = {img:[],
+            num:0
+    }
+
+    handleNum = (e) => {
+        this.setState({
+            num: e.target.value
+        })
+    }
+
+    handleSubmit = (e) => {
+        e.preventDefault()
+        this.getMultiple()
+
+    }
+
+   getMultiple = async (e) => {
+      let { num } = this.state
+
+       try{
+        let res = await axios.get(`https://cors-anywhere.herokuapp.com/https://dog.ceo/api/breeds/image/random/${num}`)
+        this.setState({img:res.data.message})
+       }catch(err){
+           console.log(err)
+       }
+   }
+
+    getRandomDog = async () => {
+        try{
+        let res = await axios.get(`https://cors-anywhere.herokuapp.com/https://dog.ceo/api/breeds/image/random/`)
+        this.setState( {img : res.data.message} )      
+        }catch(error){          
+            console.log(error)
+        }
+    }
+    
+    render(){
+        let dogs = this.state.img.map((dog)=>{  
+            //debugger        
+            return<DisplayDog img={dog} key={dog}/>
```

Comment:
The key should usually be a number, you can do this by adding an extra argument to your map method next to `dog`, so it should be `(dog, i)` then you can get the index and add it as your key